package/MDAnalysis/analysis/hole2/templates.py

```diff
@@ -23,13 +23,10 @@
 
 """HOLE Analysis --- :mod:`MDAnalysis.analysis.hole2.templates`
 =====================================================================================
-
```

Comment:
Thanks for adding back the newlines in `hole2.__init__` however there are still removed newlines in some of the other files, including here. Could you please address these?

Reply:
@IAlibay According to this https://github.com/MDAnalysis/mdanalysis/issues/2786#issuecomment-648518867
i have fixed the all issues and now package/doc/sphinx/source/documentation_pages/analysis/hole2.rst file also edited . i think all changes that was required are fixed now . please review it again.